submissions/Oleksii_Drohachov/Frogger_task/app.js

```diff
@@ -0,0 +1,182 @@
+
+const tileWidth = 100;
+const tileHeight = 83;
+const numberOfRows = 6;
+const numberOfColumns = 5;
+const fieldWidth = numberOfColumns * tileWidth;
+const fieldHeight = numberOfRows * tileHeight;
+const fieldBottomGap = 100;
+const emenyRowsModifier = 0.66;
+
+const firstEnemyRow = Math.round(tileHeight * emenyRowsModifier);
+const secondEnemyRow = Math.round(tileHeight + tileHeight * emenyRowsModifier);
+const thirdEnemyRow = Math.round(tileHeight * 2 + tileHeight * emenyRowsModifier);
+
+const entityWidth = 80;
+const entityHeight = 70;
+
+const playerStartingPositionX = tileWidth * 2; 
+const playerStartingPositionY = fieldHeight - fieldBottomGap;
+
+
+const ScoreBoard = function(score, scoreMax) {
+  this.score = score;
+  this.scoreMax = scoreMax;
+}
+
+ScoreBoard.prototype.update = function () {
+  scoreTitle.textContent = `Current score is: ${(this.score += 1)}`;
+
+  if (this.score > this.scoreMax) {
+    this.scoreMax = this.score;
+    maxScoreTitle.textContent = `Your max score is: ${this.scoreMax}`;
+  }
+};
+
+const scoreBoard = new ScoreBoard(0, 0);
+
+
+const Entity = function (xAxisPosition, yAxisPosition, sprite) {
+  this.xAxisPosition = xAxisPosition;
+  this.yAxisPosition = yAxisPosition;
+  this.width = entityWidth;
+  this.height = entityHeight;
+  this.sprite = sprite;
+}
+
+const Player = function (xAxisPosition, yAxisPosition, sprite, width, height) {
+  Entity.call(this, xAxisPosition, yAxisPosition, sprite, width, height)
+  this.moveXAxis = tileWidth;
+  this.moveYAxis = tileHeight;
+};
+
+Player.prototype = Object.create(Entity.prototype);
+
+Player.prototype.update = function () {
+  if (player.yAxisPosition < 0) {
```

Comment:
Why do you refer to a particular `Player` instance here?
Looks weird when you obviously know the proper way of referring to object's properties.